test-parachains/adder/collator/src/main.rs

```diff
@@ -31,7 +31,7 @@ use std::collections::HashMap;
 use std::sync::Arc;
 
 use adder::{HeadData as AdderHead, BlockData as AdderBody};
-use substrate_primitives::ed25519::Pair;
+use substrate_primitives::{Pair as _Pair, ed25519::Pair};
```

Comment:
What is `_Pair` used for here?

Reply:
Trait

Reply:
Why not `PairT` as we do it with `BlockT` or `HeaderT`.